ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -1287,23 +1287,30 @@ const triggerSet: TriggerSet<Data> = {
       delaySeconds: 0.2,
       suppressSeconds: 1,
       infoText: (data, _matches, output) => {
-        if (data.mustardBombTargets.includes(data.me))
-          return output.passDebuff!();
-        else if (!data.kindlingCauldronTargets.includes(data.me))
+        if (data.mustardBombTargets.includes(data.me)) {
+          const safePlayers = data.party.partyNames.filter((m) =>
+            !data.kindlingCauldronTargets.includes(m) &&
+            !data.mustardBombTargets.includes(m)
+          );
+          const fromStr = safePlayers.map((m) => data.party.member(m).nick).join(', ');
+
+          return output.passDebuff!({ from: fromStr });
```

Comment:
```suggestion
          return output.passDebuff!({ to: toStr });
```
This is a nitpick, but the Mustard Bomb targets are passing the debuff *to* the players that weren't hit by Kindling Cauldron, not picking it up from them.

Reply:
Yea. I my mind I was thinking "who can take this 'from' you" I can rename it.

Reply:
using to makes the code read better, now that I've looked at it. Thanks!